src/ComponentBase.ts

```diff
@@ -391,6 +399,40 @@ export abstract class ComponentBase<P extends React.Props<any>, S extends Object
         });
     }
 
+    /**
+     * search Subscription "keyPropertyName" in Component props(this.props)
+     *
+     * @private
+     * @param {Readonly<P>} props
+     * @param {string} keyPropertyName
+     *
+     * @return {string}
+     */
+    private _findKeyPropertyName(props: Readonly<P>, keyPropertyName: string): string {
```

Comment:
I'm making another change (just went up in PR) that should allow doing ```keyof P``` here and below

Reply:
Should I decline this PR?